Exit code: 1
Stdout:

Stderr:
Traceback (most recent call last):
  File "/Users/valeriaenriquezlimon/Documents/meta-h-generator/llm-metaheuristics/outputs-results/ollama_output_Rastrigin(6)_20250107_202751/execution_iteration_5.py", line 39, in <module>
    met.run()
  File "/Users/valeriaenriquezlimon/Documents/meta-h-generator/llm-metaheuristics/metaheuristic.py", line 144, in run
    self.apply_search_operator(perturbator, selector)
  File "/Users/valeriaenriquezlimon/Documents/meta-h-generator/llm-metaheuristics/metaheuristic.py", line 100, in apply_search_operator
    exec('Operators.' + operator_name + '(self.pop,' + operator_params)
  File "<string>", line 1, in <module>
TypeError: central_force_dynamic() got an unexpected keyword argument 'distribution'
